@xen-orchestra/backups/_cleanVm.js

```diff
@@ -117,7 +117,15 @@ exports.cleanVm = async function cleanVm(vmDir, { remove, merge, onLog = noop })
   const vhdParents = { __proto__: null }
   const vhdChildren = { __proto__: null }
 
-  const vhdsList = await listVhds(handler, vmDir)
+  let vhdsList
+  try {
+    vhdsList = await listVhds(handler, vmDir)
+  } catch (error) {
+    if (error?.code !== 'ENOENT') {
+      throw error
+    }
+    return
+  }
```

Comment:
Should be in another commit.

Reply:
The behavior change was done in this commit